docs/site/Model.md

```diff
@@ -930,18 +955,25 @@ class Product extends Entity {
     name: 'name',
     description: "The product's common name.",
     type: 'string',
-    // Specify the JSON validation rules here
+    // JSON validation rules
     jsonSchema: {
-      maxLength: 30,
       minLength: 10,
-      errorMessage:
-        'name must be at least 10 characters and maximum 30 characters',
+      maxLength: 30,
+      errorMessage: {
+        // Corresponding error messages
+        minLength: 'name should be at least 10 characters',
+        maxLength: 'name should not exceed 30 characters',
+      },
     },
   })
   public name: string;
 }
 ```
 
+Note: You can also specify a string instead of an object as a value for
+`errorMessage`. In this case the same string will be used as the error message
+for all the errors.
```

Comment:
Can you please add an example how to use a single `errorMessage` field? (See also @jannyHou comment above). IMO, the second example does not have to repeat the entire model/property definition, just show the value for `jsonSchema` field.